Formula/g/go-parquet-tools.rb

```diff
@@ -21,15 +21,14 @@ def install
       -s -w
       -X github.com/hangxie/parquet-tools/cmd.version=v#{version}
       -X github.com/hangxie/parquet-tools/cmd.build=#{time.iso8601}
-      -X github.com/hangxie/parquet-tools/cmd.gitHash=Deprecated
       -X github.com/hangxie/parquet-tools/cmd.source=Homebrew
     ]
     system "go", "build", *std_go_args(ldflags:, output: bin/"parquet-tools")
   end
 
   test do
     resource("test-parquet") do
-      url "https://github.com/hangxie/parquet-tools/raw/v1.25.10/testdata/good.parquet"
+      url "https://github.com/hangxie/parquet-tools/raw/v1.25.12/testdata/good.parquet"
```

Comment:
looks like this file never changed, you can just switched to using the sha one rather than the versioned url.
```suggestion
      url "https://github.com/hangxie/parquet-tools/raw/950d21759ff3bd398d2432d10243e1bace3502c5/testdata/good.parquet"
```

Reply:
Makes sense, addressed.